client/landing/gutenboarding/components/plans-button/index.tsx

```diff
@@ -23,16 +23,16 @@ import './style.scss';
 
 const PlansButton: React.FunctionComponent< Button.ButtonProps > = ( { ...buttonProps } ) => {
 	const { __ } = useI18n();
-	const hasPaidDomain = useSelect( ( select ) => select( ONBOARD_STORE ).hasPaidDomain() );
 
 	// mobile first to match SCSS media query https://github.com/Automattic/wp-calypso/pull/41471#discussion_r415678275
 	const isDesktop = useViewportMatch( 'mobile', '>=' );
 
-	const planLabel = sprintf(
-		/* translators: Button label where %s is the WordPress.com plan name (eg: Free, Personal, Premium, Business) */
-		__( '%s Plan' ),
-		getPlan( hasPaidDomain ? plans.PLAN_PREMIUM : plans.PLAN_FREE ).getTitle()
-	);
+	const hasPaidDomain = useSelect( ( select ) => select( ONBOARD_STORE ).hasPaidDomain() );
+	const planPath = usePlanRouteParam();
+	const plan = getPlanSlugByPath( planPath ) || ( hasPaidDomain ? defaultPaidPlan : freePlan );
+
+	/* translators: Button label where %s is the WordPress.com plan name (eg: Free, Personal, Premium, Business) */
+	const planLabel = sprintf( __( '%s Plan' ), getPlanTitle( plan ) );
```

Comment:
I'm not sure about this translation. It's not consistent with what we have over at WordPress.com/plans (both the logged-in page and logged-out marketing pages)

Many locales translation the plan names (_Free_, _Business_, and so on). I'm not arguing for one over the other, but for brand consistency, my guess is that we should describe the plans uniformly.

For example, take Japanese:

https://ja.wordpress.com/pricing/

and in Gutenboarding

<img width="234" alt="Screen Shot 2020-05-01 at 10 56 03 am" src="https://user-images.githubusercontent.com/6458278/80772546-7c9b5300-8b9a-11ea-8bf4-3e1789b3c3c0.png">

Same for German, which only translations "Personal"

<img width="603" alt="Screen Shot 2020-05-01 at 10 58 56 am" src="https://user-images.githubusercontent.com/6458278/80772678-e1ef4400-8b9a-11ea-93e6-1af26891dff0.png">

We already have 100% translation coverage for:

[Business Plan](https://translate.wordpress.com/projects/wpcom/-all-translated/84085/)
[Personal Plan](https://translate.wordpress.com/projects/wpcom/-all-translated/242357/)
[Premium Plan](https://translate.wordpress.com/projects/wpcom/-all-translated/114031/)
Strangely "eCommerce Plan" doesn't yet exist, but there's [eCommerce](https://translate.wordpress.com/projects/wpcom/-all-translated/83441/)

Reply:
If I understood correctly, you suggested taking "Plan" word out of the header? I think it would be important to keep "Plan" word in the header for clarity (_"Premium" what?_). When plans are side-by-side on a grid, it's more clear that "personal" refers to a plan, but in the header it could refer e.g. to automatically assigned vertical. (@dubielzyk)

I think it's ok to merge this PR without changes to this I think and we can log follow-ups; generally we can bend the rest of the system for designs, vs try to bend designs for the code we happen to have.

Reply:
Yeah, let's make the least amount of copy changes for now and I'll follow-up where we should override. We wanna be as consistent with the previous designs as possible

Reply:
> If I understood correctly, you suggested taking "Plan" word out of the header?

No, just highlighting that we need to be consistent. 

We're introducing a new i18n string with `__( '%s Plan' )`. 

Because of this, I think we need to make sure that we use the same nomenclature across our application including marketing pages, or that we follow the existing implementation.

Reply:
Fair point. I noted down consistently translating plans at https://github.com/Automattic/wp-calypso/issues/37665